Makefile

```diff
@@ -1,5 +1,5 @@
-REPO_REVISION=20fab815f075645cdb7f320f5addbc1790b18dff
-VERSION=0.2.2.1-$(shell date '+%Y%m%d')
+REPO_REVISION=6871b44b5735645eced8fa5737be4c349862b4b3
+VERSION=0.2.2.2-$(shell date '+%Y%m%d')
```

Comment:
The matching version for this revision is `0.2.8.0`

Reply:
Version updated.